pages/admin/dashboard/tools.tsx

```diff
@@ -245,7 +245,7 @@ const Tools: React.FC<ToolsProps> = ({ selectedTab }) => {
                                     ? new Date(tool.updatedAt)
                                     : new Date()
                             }
-                            module={tool.linkedModuleID !== null}
+                            module={tool.linkedModuleID || false}
```

Comment:
Why the choice of "false" instead of "" or null?

Reply:
Tru i think null actually makes more sense here (latest commit)